### ISSUE TITLE
DbStarter fails when db.tcpServer param is specified

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#260]
```
Using a Servlet Listener to Start and Stop a Database

DbStarter can also start the TCP server, but if enable the param db.tcpServer 
the listener fails.
The listener init connection parameters and then to connect, but all before 
parse and start server, if connection fails server is not startup.

The patch is move the 2 code line of connection after the start was server.
```

Original issue reported on code.google.com by `nikolas....@finantix.com` on 26 Nov 2010 at 12:27

**Comment by GoogleCodeExporter** [comment]
```
sorry for bad english.
Steps:
- Listener initialize connection parameters.
- Listener try connect to database using parameters.
Database is not up, throw an exception that return.
The code to startup database never be done.

The patch is move code for connection after startup the server if db.tcpServer 
is specified.
```

Original comment by `nikolas....@finantix.com` on 26 Nov 2010 at 1:32

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Thanks for the patch! I will commit it. However, please note that connecting 
using the server URL is slower than connecting in embedded mode.

Regards,
Thomas
```

Original comment by `thomas.t...@gmail.com` on 30 Nov 2010 at 7:48
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.148
```

Original comment by `thomas.t...@gmail.com` on 12 Dec 2010 at 12:34
- Changed state: **Fixed**
